.vscode/settings.json

```diff
@@ -4,4 +4,7 @@
         "rust-analyzer"
     ],
     "rust-analyzer.check.command": "clippy",
+    "rust-analyzer.linkedProjects": [
+        "./implants/lib/transport/Cargo.toml"
```

Comment:
uhhhhh, methinks this was fro that one time we hit the problem and didn't know what it was.